README.md

```diff
@@ -50,12 +50,13 @@ If you want to simulate that your cache server is slow at incoming network
 upstream:
```

Comment:
I was a little too liberal with the updates to just `toxic` (mos people don't care about the distinction), thx